File: CPP/Program/Kurir.cpp
#include "Pengiriman.cpp"

// Kelas Kurir turunan dari Pengiriman
class Kurir : public Pengiriman{
    private:
        string namaKurir;           // Nama kurir
        string statusPengiriman;    // Status pengiriman
        string noHP;                // Nomor HP kurir

    public:
        Kurir(){} // Konstruktor default

        // Setter nama kurir
        void setNamaKurir(string nama){
            this->namaKurir = nama;
        }

        // Getter nama kurir
        string getNamaKurir(){
            return this->namaKurir;
        }

        // Setter status pengiriman
        void setStatusPengiriman(string status){
            this->statusPengiriman = status;
        }

        // Getter status pengiriman
        string getStatusPengiriman(){
            return this->statusPengiriman;
        }

        // Setter nomor HP kurir
        void setNoHP(string noHP){
            this->noHP = noHP;
        }

        // Getter nomor HP kurir
        string getNoHP(){
            return this->noHP;
        }
};

File: Java/Program/Kurir.java
public class Kurir extends Pengiriman {
    private String namaKurir;           // Nama kurir
    private String statusPengiriman;    // Status pengiriman
    private String noHP;                // Nomor HP kurir

    public Kurir() {} // Konstruktor default

    // Setter nama kurir
    public void setNamaKurir(String nama) {
        this.namaKurir = nama;
    }

    // Getter nama kurir
    public String getNamaKurir() {
        return namaKurir;
    }

    // Setter status pengiriman
    public void setStatusPengiriman(String status) {
        this.statusPengiriman = status;
    }

    // Getter status pengiriman
    public String getStatusPengiriman() {
        return statusPengiriman;
    }

    // Setter nomor HP kurir
    public void setNoHP(String noHP) {
        this.noHP = noHP;
    }

    // Getter nomor HP kurir
    public String getNoHP() {
        return noHP;
    }
}
